index.css

```diff
@@ -0,0 +1,1068 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background: gray;
```

Comment:
- [ ] Kindly change the background color of the navbar to a different kind of gray so it is different than the article right below (like the previously TSE asked).

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/90390174-44dcec00-e061-11ea-88d9-8a694386f2cf.png) |![image](https://user-images.githubusercontent.com/61479648/90390213-56be8f00-e061-11ea-8f5f-d5fe941934f7.png)|

![image](https://user-images.githubusercontent.com/61479648/90390484-c5035180-e061-11ea-852e-9bbd063dd488.png)

Reply:
ok done